common-rest/src/test/java/org/uniprot/api/rest/service/query/UniProtQueryProcessorTest.java

```diff
@@ -236,6 +236,14 @@ void handleCamelCaseFieldSearch() {
         assertThat(processedQuery, is("field:thing_value"));
     }
 
+    @Test
+    void handleGreekSymbolValueSearch() {
+        String query = "field:other_value"+"\u03B1";
+        when(searchFieldConfig.getSearchFieldNames()).thenReturn(Set.of("field"));
+        String processedQuery = processor.processQuery(query);
+        assertThat(processedQuery, is("field:other_valuealpha"));
+    }
+
```

Comment:
Add more positive and negative unit tests.
Add integration test at controller level as well.